actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -93,6 +93,43 @@ def assert_enqueued_emails(number, &block)
       assert_enqueued_jobs number, only: [ ActionMailer::DeliveryJob, ActionMailer::Parameterized::DeliveryJob ], &block
     end
 
+    # Asserts that a specific email has been enqueued, optionally
+    # also matching arguments.
+    #
+    #   def test_emails
+    #     ContactMailer.welcome.deliver_later
+    #     assert_enqueued_email_with ContactMailer, :welcome
+    #   end
```

Comment:
Given that the description above alludes to passing `args`, we should include an example of that.